SDAPackageWithVectorField/SDAWithVectorField/moving_obstacle.py

```diff
@@ -21,13 +21,16 @@ def __init__(self, point, radius):
         :param radius: The radius of the obstacle
         :type radius: Float
         """
-        super(MovingObstacle, self).__init__(point, Constants.MOVING_OBSTACLE_SAFETY_RADIUS)
+        super(MovingObstacle, self).__init__(point)
 
         self.radius = radius
+        self.type = "moving"
```

Comment:
As we talked about in class, please you class type as the distinguisher between stationary and moving, not a string.